generated/beats/fields.ecs.yml

```diff
@@ -1122,6 +1122,7 @@
         norms: false
         default_field: false
       description: The stack trace of this error in plain text.
+      index: false
```

Comment:
I wonder what the final template will look like here? Will it still index the `text` part but skip keyword? Or does it apply to both?

Reply:
Ah good catch. So I tried testing a similar scenario in ES where the outer field `error.stack_trace` has `index: false` but the inner one `error.stack_trace.text` does not have that. Looks like `error.stack_trace` is not searchable as expected, but `error.stack_trace.text` will be. The template that ecs generates for this schema is here: https://github.com/elastic/ecs/blob/master/generated/elasticsearch/7/template.json#L650-L661 which hasn't changed from the edits I've made here.

@webmat is it intentional to have the `error.stack_trace.text` be searchable while the `error.stack_trace.keyword` is not searchable?

Reply:
Not intentional, no.

Also, this highlights the fact that despite flagging a few fields as `index: False` in ECS, this never made it to the Beats mapping. Oops 🙊. Changing this now would cause an expected breaking change for Beats, IMO.

The addition of this feature makes total sense, but we'll need to decide what we do with this Beats change.

Reply:
I think @andrewkroh would be your best bet

Reply:
In elastic/beats `log.original` and `event.original` are used, but I didn't find anything that would be querying them. So I think the only effect would be if users had custom queries (like a regexp query on the `event.original`).

And nothing in beats populates `error.stack_trace` from what I could find. @simitt would anything in APM be affected by setting `index: false` for `error.stack_trace` (like does anything query error.stack_trace)? I imagine anything that needs to search it would use error.stack_trace.text`.

Reply:
Thanks for the ping; APM does not use `event.original`, `log.original` nor `error.stack_trace` at the moment.

Reply:
Thanks for taking a look @andrewkroh and @simitt. Is there anything else we could do to mitigate the breaking change risk?

Reply:
Any other concerns with merging this PR?